main.py

```diff
@@ -1,12 +1,210 @@
 import requests
+from video_store import Videostore
 
 URL = "http://127.0.0.1:5000"
-BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
+VIDEO_STORE_URL = "https://retro-video-store-api.herokuapp.com"
 
 def main():
-    print("WELCOME TO RETRO VIDEO STORE")
-    pass
+    print("################################################")
+    print("******** WELCOME TO RETRO VIDEO STORE **********")
+    print("################################################")
 
 
 if __name__ == "__main__":
-    main()
\ No newline at end of file
+    main()
+
+def list_options():
+    options = {
+        "1": "Create a video",
+        "2": "Update selected video",
+        "3": "Delete selected video",
+        "4": "List all videos",
+        "5": "Select a video", 
+
+        "6": "Create a customer",
+        "7": "Update selected customer",
+        "8": "Delete selected customer",
+        "9": "List all customers",
+        "10": "Select a customer",
+
+        "11": "Check out a video to a customer",
+        "12": "Check out a video to a customer",
```

Comment:
There's a little typo here that confused me when I ran the program
```suggestion
        "12": "Check in a video to a customer",
```